papers/bcr-0004-bc32.md

```diff
@@ -0,0 +1,51 @@
+# The BC32 Data Encoding Format
+## BCR-0004
+
+**© 2020 Blockchain Commons**
+
+Author: Wolf McNally<br/>
+Date: May 2, 2020
+
+---
+
+### Introduction
+
+The [Bech32] checksummed base32 format was introduced to transport segregated witness signatures for Bitcoin. It has several desirable qualities:
+
+* The payload and checksum data use a limited character set compatible with both URI syntax and QR code alphanumeric encoding mode.
```

Comment:
encoding mode with its built-in compression.

Reply:
It's not really "compression", it's merely higher efficiency than encoding generic ASCII or base64. I'll add a note to this effect.